Doc/library/sqlite3.rst

```diff
@@ -1069,6 +1089,7 @@ Connection objects
          Added the ``sqlite3.enable_load_extension`` auditing event.
 
       .. testsetup:: sqlite3.loadext
+         :skipif: True
```

Comment:
What's the point of having a test if it's always skipped? Remove it if it always fail?